src/System.Security.Cryptography.Csp/tests/PasswordDeriveBytesTests.cs

```diff
@@ -276,6 +276,19 @@ public static void GetBytes_StreamLike()
         }
 
         [Fact]
+        public static void GetBytes_Boundary()
+        {
+            using (var deriveBytes = new PasswordDeriveBytes(TestPassword, s_testSalt))
+            {
+                // Boundary case success
+                deriveBytes.GetBytes(1000 * 20);
+
+                // Boundary case failure
+                Assert.Throws<CryptographicException>(() => deriveBytes.GetBytes(1001 * 20));
```

Comment:
Doesn't it save the block count as a field, so only one needs to be called here?